src/mame/drivers/spg2xx_pdc.cpp

```diff
@@ -70,22 +79,29 @@ void spg2xx_pdc100_game_state::pdc100(machine_config &config)
 }
 
 ROM_START( pdc100 )
-	ROM_REGION( 0x8000000, "maincpu", ROMREGION_ERASE00 )
+	ROM_REGION( 0x4000000, "maincpu", ROMREGION_ERASE00 )
 	// only 1st half of this is used "Jumper resistor (0 ohm) that short A25 to ground"
 	// 2nd half just contains what seems to be random garbage
-	ROM_LOAD16_WORD_SWAP( "pdc100.bin", 0x000000, 0x8000000, CRC(57285b49) SHA1(cfb4be7877ec263d24063a004c56985db5c0f4e2) )
+	ROM_LOAD16_WORD_SWAP( "pdc100.bin", 0x000000, 0x4000000, CRC(57285b49) SHA1(cfb4be7877ec263d24063a004c56985db5c0f4e2) )
```

Comment:
Can you update the hashes please?

Reply:
the hashes don't need updating, the file hasn't changed.

Reply:
Except you changed the size of the file, and got rid of the second half filled with garbage, so of course the hashes are going to need updating.

Reply:
Except you changed the size and got rid of the entire second half, so the hashes will change anyway.

Reply:
No, I haven't changed the size of the file, read the whole code.